mobile/src/navigations/root/root.tsx

```diff
@@ -82,17 +79,10 @@ const Root: React.FC = () => {
             />
         ),
         main: (
-            <>
-                <RootStack.Screen
-                    name={RootScreenName.MAIN_ROOT_ROUTE}
-                    component={
-                        role === UserRole.TALENT
-                            ? TalentBottomTabNavigator
-                            : EmployerBottomTabNavigator
-                    }
-                />
-                <RootStack.Screen name={RootScreenName.CHAT} component={Chat} />
-            </>
+            <RootStack.Screen
+                name={RootScreenName.MAIN_ROOT_ROUTE}
+                component={ChatNavigator}
```

Comment:
Why main root is chat navigator?

Reply:
There was bottom tab navigator, but since I need to remove bottom tabs in the chat screen and company info screen. I made it like is written in the [documentation ](https://reactnavigation.org/docs/hiding-tabbar-in-screens/)